QUICKSTART.md

```diff
@@ -1,42 +1,67 @@
+summary: Learn how to deploy, monitor, test and stop a TiDB cluster.
 category: quick start
 ---
 
 # TiDB Quick Start Guide
 
-## About TiDB
+## Before you begin
 
-TiDB (The pronunciation is: /'taɪdiːbi:/ tai-D-B, etymology: titanium) is an open-source distributed scalable Hybrid Transactional and Analytical Processing (HTAP) database. It features infinite horizontal scalability, strong consistency, and high availability. TiDB is MySQL compatible and serves as a one-stop data warehouse for both OLTP (Online Transactional Processing) and OLAP (Online Analytical Processing) workloads.
+This guide introduces you the quickest way to deploy a TiDB cluster locally - using Docker Compose. Once you've installed [Docker Compose](https://docs.docker.com/compose/install/), [Git](https://git-scm.com/downloads) and [MySQL Server](https://dev.mysql.com/downloads/mysql/), you are all set.
 
-## About this guide
+> **Warning:** Running TiDB in Docker involves risks and is strongly discouraged for production application. For the production environment, it is recommended to [deploy TiDB using Ansible](op-guide/ansible-deployment.md).
 
-This guide outlines how to perform a quick deployment of a TiDB cluster using TiDB-Ansible and walks you through the basic TiDB operations and administrations.
+## Deploy a TiDB cluster
 
-## Deploy the TiDB cluster
+First, open a terminal and enter the following commands:
 
-This section describes how to deploy a TiDB cluster. A TiDB cluster consists of different components: TiDB servers, TiKV servers, and Placement Driver (PD) servers.
+1. To download `tidb-docker-compose`:
 
-The architecture is as follows:
+    ```bash
+    git clone https://github.com/pingcap/tidb-docker-compose.git
+    ```
+
+2. To get the latest TiDB Docker images and start the cluster:
+
+    ```bash
+    cd tidb-docker-compose && docker-compose pull 
+    docker-compose up -d
+    ```
+
+3. Make sure that you have enabled the MySQL Server service in order to access the cluster:
+
+    ```bash
+    mysql -h 127.0.0.1 -P 4000 -u root
+    ```
+
+## Monitor the cluster
+
+After your machine successfully connects to MySQL Server on `127.0.0.1`, you can monitor real-time activities in the TiDB cluster with:
+
+1. The [Grafana monitoring interface](op-guide/monitor-overview.md/#about-grafana-in-tidb):
 
-![TiDB Architecture](media/tidb-architecture.png)
+    - Default address: <http://localhost:3000>
+    - Default account name: admin
+    - Default password: admin
 
-To quickly deploy a TiDB testing cluster, see [Deploy TiDB Using Docker Compose](op-guide/docker-compose.md).
+2. The [cluster data visualization interface](https://github.com/pingcap/tidb-vision): 
 
-## Try TiDB
+    - Default address: <http://localhost:8010>
 
-This section describes some basic CRUD operations in TiDB.
+## Test the cluster
+
+This section demonstrates some basic CRUD operations of TiDB in the terminal.
 
 ### Create, show, and drop a database
```

Comment:
Please add another .md file for the rest of the part. Let's keep the quick start part as simple as possible.